.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+sudo: false
+python:
+    - pypy
+    - 2.6
+    - 2.7
+    - 3.2
+    - 3.3
+    - 3.4
+install:
+    - pip install . --use-mirrors
```

Comment:
`--use-mirrors` is deprecated and pip ignores it.

This is also an excellent opportunity to update this PR so we can see if Travis will start building it.